### ISSUE TITLE
Show platform

### ISSUE THREAD

**Issue by pstray** [jonkristian/entur-card#30]
Is it possible to show which platform a train is leaving from?  My train isn't always on the same track each time, at least for Oslo S (and also my local station), so it would be nice to have that info on the card.

**Comment by jonkristian** [comment]
@pstray correct me if I'm wrong, but isn't platform in the name of the route, like L1, L2? (you can get the friendly name of the sensor if you don't override the name). I can't see anything else regarding platform being exposed in the sensor, I can only work with what is exposed, so if that data doesn't exist I think this issue should be reported on the entur component. Just need a confirmation here, if I am wrong please provide which attribute you're looking at and how you want it presented :-)

**Comment by pstray** [comment]
No, the platform is not in the name. L1, L2, R10, and such is the line name, not the platform.

Not sure if the platform is exposed, but it is definitely in the source data as the sensor can split into platforms.

**Comment by jonkristian** [comment]
Can you provide more details, examples of platform names your train is leaving from, train line id etc..? Also, the info that you're after, can you find it at the [stop place registry](https://stoppested.entur.org) or [entur.no](https://entur.no/) ?

The logic might be in the entur component, but if the data isn't exposed in the sensor(s) there's not much I can do :disappointed: and it's something that should be handled in the entur component, so it should be reported [here](https://github.com/home-assistant/core/issues?q=is%3Aissue+label%3A%22integration%3A+entur_public_transport%22+is%3Aopen).

That being said, if you look at any of the sensors and find some data to work with and explain how you'd like it to behave, I'd be more than happy to help with figuring out a way, for instance maybe a template sensor with filtering.
